### DISCUSSION TITLE
Bulk Submit Listing Multiple User Accounts As The Reporter

### DISCUSSION THREAD

**Discussion by siberian-devsky** [pycontribs/jira#1357] [Q&A]
### Summary
Hi. I notice that when I use the Python-Jira library’s client.create_issues() method that the first issue lists **_User A_** as the reporter but all subsequent entries are submitted by our Generic Jira Admin account (henceforth **_User B_**).

The library documentation makes no mention of this behavior. Is this to be expected?

### Expected Behavior
I would assume that all issues submitted as part of a bulk operation would record as the reporter the user account used to initialize the Jira Client connection (see below steps to reproduce) i.e. all issues list the reporter as **_User A_**

### Actual Behavior
The first issue created lists the reporter as **_User A_** but all subsequent issues list the reporter as **_User B_**

### Steps to replicate:
1. Initialize the Jira Client using token_auth and my API key - **_User A_**
2. Gather data and build a list of objects to send to create_issues()
3. Send the data
4. Navigate to Jira and note that only the first of the bulk issues lists **_User A_** as the reporter. All the subsequent issues list **_User B_** 

### Versions
Python Interpreter:  3.10.4
jira-python: 3.1.1
OS:  WIn 10 64-bit

**Comment by adehad** [comment]
Hi @jmartin111, thanks for this.

This sounds like it could be a subtlety in the Jira API. 

I wonder if the `reporter` needs to be passed in explicitly to get this to work?

Don't suppose you could tell me if this is Jira Server or Cloud (FYI we have a bug report template here: https://github.com/pycontribs/jira/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml)

**Comment by siberian-devsky** [comment]
Hi Adel,

This is Jura server

Best,
Jeff

**Comment by siberian-devsky** [comment]
Also,

I wouldn't think that the submitter should be to be passed in. It's not
even a field one can fill out when creating a ticket. Jira simply uses the
current user logged in. Or, in this case, the user who initialized the API.

Best,
Jeff
